tests/unit/processor/string_splitter/test_string_splitter.py

```diff
@@ -0,0 +1,60 @@
+# pylint: disable=missing-docstring
+import pytest
+from logprep.processor.base.exceptions import ProcessingWarning
+from tests.unit.processor.base import BaseProcessorTestCase
+
+
+test_cases = [
+    (
+        "splits without delimeter on whitespace",
+        {
+            "filter": "message",
+            "string_splitter": {"source_fields": ["message"], "target_field": "result"},
+        },
+        {"message": "this is the message"},
+        {"message": "this is the message", "result": ["this", "is", "the", "message"]},
+    )
+]  # testcase, rule, event, expected
```

Comment:
I think that we should also add a test case that uses any other delimiter that is not a whitespace.